lib/crypto.js

```diff
@@ -60,7 +60,8 @@ const {
 const {
   createSecretKey,
   createPublicKey,
-  createPrivateKey
+  createPrivateKey,
+  KeyObject
```

Comment:
trailing `,`, please

Reply:
- The KeyObject as a Class is [already documented](https://nodejs.org/api/crypto.html#crypto_class_keyobject)
- The file is already mixed trailing / non-trailing, this one didn't change the particular const declaration as you see. Do you want me to align the complete file then?

Reply:
legacy is non-trailing, we are moving to trailing, but not as a bulk change. You can see why in the diff above... there is a diff to the `createPrivateKey` line which was only necessary because you had to add an extra member. By adding the trailing `,` now on the line you adding, the next person to add something won't have touch unrelated code, they can add just the single line that they want.

Reply:
Sure. Understood.